### DISCUSSION TITLE
VS Code extension - unauthorized for business license of copilot

### DISCUSSION THREAD

**Discussion by rafek1241** [community/community#47524] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I used new license of business version for github copilot and when I created PAT on my github user account:
```
[ERROR] [default] [2023-02-15T08:57:10.139Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
[INFO] [auth] [2023-02-15T09:00:31.256Z] Invalid copilot token: missing token: 404 
```
even though I'm logged in in a **VSCodium**.

What should I do in this scenario? I couldn't find any suitable tutorial and I tried to create PAT on my own to give it more priviliges, nothing worked on. Lastly, I changed my associated organization security setting regarding PAT to be able to connect. Nothing changed.

**For jetbrains plugin organization license works perfectly fine.**

**Comment by unknowVariable** [comment]
It looks like you are encountering an issue with the GitHub Copilot extension for Visual Studio Code. Based on the error message you provided, it seems like the extension is having trouble connecting to the GitHub server using the Personal Access Token (PAT) you generated.

Here are a few steps you can try to resolve the issue:

1. Check your GitHub Copilot license: Make sure that your license is valid and up-to-date. You may also want to check if the license is associated with your GitHub account.

2. Regenerate your PAT: Try generating a new PAT with the appropriate permissions and use it to authenticate the extension.

3.  Verify your network settings: Make sure that your network settings are configured correctly and there are no firewalls or network restrictions preventing the extension from connecting to the GitHub server.

4. Try using a different IDE: If the issue persists, you may want to try using a different IDE or code editor to see if the issue is specific to Visual Studio Code.

If none of these solutions work, you may want to contact the GitHub support team or the extension developer for further assistance.

**Reply by rafek1241** [reply]
I love this standard response. Is it ChatGPT? :)

**Comment by rafek1241** [comment, marked as answer]
For people who has the same issue:
https://github.com/VSCodium/vscodium/issues/1388
